omero/developers/Server/ObjectGraphs.txt

```diff
@@ -8,9 +8,10 @@ Model graph operations
   deleting users' rendering settings for that image, also the links to
   any datasets that the image is in. Version 5.1.0 of the OMERO server
   includes a new algorithm for determining which model objects to act
-  on. Understanding the details of the new implementation substantially
-  assists in debugging or creating server operations that act on the
-  directed graph of model objects.
+  on. Clients should be :doc:`GraphsMigration` because the previous ones
+  will be *removed* in OMERO 5.3. Understanding the details of the new
```

Comment:
5.3 to give client developers plenty of time to fix their clients before we break them completely. (Many people's scripts will need adjusting.)